packages/tools/client-debugger/client-debugger-view/src/DevtoolsView.tsx

```diff
@@ -0,0 +1,449 @@
+	 */
+	containers?: ContainerMetadata[];
+}
+
+/**
+ * View body component used by {@link DevtoolsView}.
+ */
+function View(props: ViewProps): React.ReactElement {
+	const { menuSelection, containers } = props;
+
+	let view: React.ReactElement;
+	switch (menuSelection?.type) {
+		case "telemetryMenuSelection":
+			view = <TelemetryView />;
+			break;
+		case "containerMenuSelection":
+			// eslint-disable-next-line no-case-declarations
+			const container = containers?.find((x) => x.id === menuSelection.containerId);
+			view =
+				container === undefined ? (
+					<div>Could not find a debugger for that container.</div>
+				) : (
+					<ContainerDevtoolsView containerId={menuSelection.containerId} />
+				);
+			break;
+		default:
+			view = <LandingView />;
+			break;
+	}
+
+	return (
+		<Stack.Item grow={5} styles={contentViewStyles}>
+			<div
+				id="debugger-view-content"
+				style={{ width: "100%", height: "100%", overflowY: "auto" }}
+			>
+				{view}
+			</div>
+		</Stack.Item>
+	);
+}
+
+/**
+ * {@link Menu} input props.
+ */
+interface MenuProps {
+	/**
+	 * The current menu selection (if any).
+	 */
+	currentSelection?: MenuSelection;
+
+	/**
+	 * Sets the menu selection to the specified value.
+	 *
+	 * @remarks Passing `undefined` clears the selection.
+	 */
+	setSelection(newSelection: MenuSelection | undefined): void;
+
+	/**
+	 * Set of features supported by the {@link FluidDevtools} instance being used by the page application.
+	 */
+	supportedFeatures: DevtoolsFeatureFlags;
+
+	/**
+	 * The set of Containers to offer as selection options.
+	 */
+	containers?: ContainerMetadata[];
+}
+
+/**
+ * Menu component for {@link DevtoolsView}.
+ */
+function Menu(props: MenuProps): React.ReactElement {
+	const { currentSelection, setSelection, supportedFeatures, containers } = props;
+
+	function onContainerClicked(id: string): void {
+		setSelection({ type: "containerMenuSelection", containerId: id });
+	}
+
+	function onTelemetryClicked(): void {
+		setSelection({ type: "telemetryMenuSelection" });
+	}
+
+	const menuSections: React.ReactElement[] = [];
+
+	menuSections.push(
+		<ContainersMenuSection
+			containers={containers}
+			currentContainerSelection={
+				currentSelection?.type === "containerMenuSelection"
+					? currentSelection.containerId
+					: undefined
+			}
+			selectContainer={onContainerClicked}
+		/>,
+	);
+
+	// Display the Telemetry menu section only if the corresponding Devtools instance supports telemetry messaging.
+	if (supportedFeatures[DevtoolsFeature.Telemetry] === true) {
+		menuSections.push(
+			<MenuSection header="Telemetry" key="telemetry-menu-section">
+				<MenuItem
+					isActive={currentSelection?.type === "telemetryMenuSelection"}
+					text="See Telemetry"
+					onClick={onTelemetryClicked}
+				/>
+			</MenuSection>,
+		);
+	}
+
+	return (
+		<Stack.Item grow={1} styles={menuStyles}>
+			{/* TODO: button to refresh list of containers */}
```

Comment:
I think move the comment to the ContainersMenuSection for clarity of where it needs to be implemented.